front-end/src/ui-components/ScrollToTopButton.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
```

Comment:
The name of the file is `ScrollToTopBottom` should be changed to remove the "Bottom" part.

Reply:
oops, I wasn't awake it seems!